webapp/src/webapp/webclient/log_area/main.cljs

```diff
@@ -86,18 +102,13 @@
          [:div {:class (str (if (= feature-ai-ask "enabled")
                               "h-terminal-container"
                               "h-full"))}
-          [tabs {:on-click #(reset! selected-tab %2)
-                 :tabs (merge
-                        (when (and (= feature-ai-ask "enabled")
-                                   connection-type-database?
-                                   is-one-connection-selected?)
-                          {:ai "AI"})
-                        {:terminal "Terminal"}
-                        (when connection-type-database?
-                          {:tabular "Tabular"}))
+          [tabs {:on-click (fn [_ value]
+                             (.setItem js/localStorage "webclient-selected-tab" value)
+                             (reset! selected-tab value))
+                 :tabs available-tabs
                  :selected-tab @selected-tab}]
           (case @selected-tab
             "AI" [terminal/main :ai ai-content]
             "Tabular" [data-grid-table/main results-heads results-body true tabular-loading?]
-            "Terminal" [terminal/main :terminal terminal-content]
+            "Logs" [terminal/main :terminal terminal-content]
```

Comment:
I'd add some refactoring to these names as well so they can reflect what they are. Also, we have a "terminal" like experience in the upcoming future, having things with their names all refactored on these changes will help that